### PR TITLE
Feature/issue 615 include mb data in import scripts

### PR DESCRIPTION
**Pull request by rasmus-storjohann-PG**

### CODE REVIEW

File: newcomers_guide/management/commands/import_newcomers_guide.py [hunk 1]
```diff
@@ -32,3 +32,5 @@ def handle(self, *args, **options):
 
         counts = ImportCounters()
         save_topics(topics, counts)
+
+# add bc or mb prefix to topic ids
```
Comment (rasmus-storjohann-PG) [inline newcomers_guide/management/commands/import_newcomers_guide.py:36]: remove

File: newcomers_guide/management/commands/generate_client_fixtures.py [hunk 2]
```diff
@@ -38,3 +38,6 @@ def handle(self, *args, **options):
 
         with open('taxonomies.ts', 'w') as file:
             file.write(generate_taxonomy_fixture(taxonomies))
+
+# add bc or mb prefix to each topic id
+# add region taxonomy term to each topic
```
Comment (rasmus-storjohann-PG) [inline newcomers_guide/management/commands/generate_client_fixtures.py:43]: remove

File: utility/prepare_deploy.sh [hunk 3]
```diff
@@ -145,41 +186,21 @@ checkForSuccess "reset database"
...
-echo "converting additional schools CSV to open referral standard..."
-mkdir -p ./open_referral_csv_files_schools
-./manage.py convert_icarol_csv ../content/additionalSchools.csv ./open_referral_csv_files_schools
-checkForSuccess "convert additional schools data into open referral standard"
-
-echo "importing additional schools open referral csv data into the database..."
-./manage.py import_open_referral_csv ./open_referral_csv_files_schools --cityLatLongs $CityLatLongs
-checkForSuccess "import additional schools open referral data into the database"
+importICarolCsvServiceData $BC211Path "${OutputDir}/openreferral/211/bc" bc
+importICarolCsvServiceData ../content/organizationAsServices.csv "${OutputDir}/openreferral/organizationsAsService" bc
```
Comment (rasmus-storjohann-PG) [inline utility/prepare_deploy.sh:190]: Add `_bc` suffix to primary keys

File: utility/prepare_deploy.sh [hunk 4]
```diff
@@ -145,14 +158,7 @@ checkForSuccess "reset database"
...
 
-echo "converting iCarol BC-211 CSV to open referral standard..."
-mkdir -p ./open_referral_csv_files
-./manage.py convert_icarol_csv $BC211Path ./open_referral_csv_files
-checkForSuccess "convert iCarol BC-211 data into open referral standard"
-
-echo "importing BC-211 open referral csv data into the database..."
-./manage.py import_open_referral_csv ./open_referral_csv_files --cityLatLongs $CityLatLongs
-checkForSuccess "import Bc-211 open referral data into the database"
+importICarolCsvServiceData $BC211Path ./open_referral_csv_files bc
```
Comment (tomy-pg) [inline utility/prepare_deploy.sh]: Nice. The helper makes it much easier to understand what is going on

**Review by tomy-pg — APPROVED** [review]
Looks good to me

File: utility/prepare_deploy.sh [hunk 5]
```diff
@@ -97,6 +113,15 @@ validateNewcomersGuidePath () {
     fi
 }
 
+validateManitobaWinPath() {
+    if [ "$manitobaWinPath" != "" ] && [! -f $manitobaWinPath ]
```
Comment (tomy-pg) [inline utility/prepare_deploy.sh:117]: missing space after square bracket causes this error:
`./utility/prepare_deploy.sh: line 117: [!: command not found`

The script still runs but crashes before any WIN data can be processed.

Change to:
`if [ "$manitobaWinPath" != "" ] && [ ! -f $manitobaWinPath ]`

**Review by tomy-pg — CHANGES_REQUESTED** [review]

File: newcomers_guide/management/commands/import_newcomers_guide.py (hunk 1 above)
Comment (rasmus-storjohann-PG) [reply newcomers_guide/management/commands/import_newcomers_guide.py:36]: done

File: newcomers_guide/management/commands/generate_client_fixtures.py (hunk 2 above)
Comment (rasmus-storjohann-PG) [reply newcomers_guide/management/commands/generate_client_fixtures.py:43]: done

File: utility/prepare_deploy.sh (hunk 5 above)
Comment (rasmus-storjohann-PG) [reply utility/prepare_deploy.sh:117]: thanks, fixed

File: utility/prepare_deploy.sh (hunk 3 above)
Comment (rasmus-storjohann-PG) [reply utility/prepare_deploy.sh:193]: No change needed, brain fart on my part

**Pull request merged**